classes/class04/homework/public/javascripts/main.js

```diff
@@ -0,0 +1,119 @@
+var loadThings = function() {
+    $('form.toggleIngredient').unbind();
+    $('form#newIngredient').unbind();
+    $('input.edit').unbind();
+    $('input.pickIngredient').unbind();
+    $('form.order').unbind();
+
+    // Toggling Ingredients
+    $('form.toggleIngredient').submit(function(event) {
+        event.preventDefault();
+
+        var $form = $(event.target).closest('form');
+        var current = $form.find('input[class="toggle"]');
+
+        if (current.val() == "In Stock") {
+            current.val("Out of Stock");
+            $.post('/toggleIngredient', {
+                id: $form.attr('id'),
+                stock: false
+            });
+        } else {
+            current.val("In Stock");
+            $.post('/toggleIngredient', {
+                id: $form.attr('id'),
+                stock: true
+            });
+        }
+    });
+
+    // Add Ingredient
+    $('form#newIngredient').submit(function(event) {
+        event.preventDefault();
+        $form = $(event.target);
```

Comment:
In most (maybe all) event handling callbacks `this === event.target`and is a more commonly used variable.